src/pudl/metadata/classes.py

```diff
@@ -2000,9 +2020,17 @@ def from_data_source_ids(
             xbrl_resources=xbrl_resources,
         )
 
-    def to_yaml(self, path: str = None) -> None:
+    def to_yaml(
+        self, path: str = None, exclude_intermediate_resources: bool = False
+    ) -> None:
         """Output database, table, and column metadata to YAML file."""
         template = _get_jinja_environment().get_template("datasette-metadata.yml.jinja")
+        if exclude_intermediate_resources:
+            [
+                resource
+                for resource in self.resources
+                if not resource.name.startswith("_")
+            ]
```

Comment:
Added this if we want to exclude intermediate assets from datasette.  I think we should include them for now so the database, data dictionary and datasette are all consistent.